src/acquisition/covidcast/database.py

```diff
@@ -318,32 +283,44 @@ def run_dbjobs(self):
             `lag` = sl.`lag`,
             `missing_value` = sl.`missing_value`,
             `missing_stderr` = sl.`missing_stderr`,
-            `missing_sample_size` = sl.`missing_sample_size` 
+            `missing_sample_size` = sl.`missing_sample_size`
     '''
 
+    # NOTE: DO NOT `TRUNCATE` THIS TABLE!  doing so will ruin the AUTO_INCREMENT counter that the history and latest tables depend on...
     signal_load_delete_processed = f'''
-        DELETE FROM `{self.load_table}` 
-        WHERE  process_status <> '{PROCESS_STATUS.LOADED}'
+        DELETE FROM `{self.load_table}`
     '''
 
-    print('signal_load_set_comp_keys:')
-    self._cursor.execute(signal_load_set_comp_keys)
-    print('signal_load_mark_batch:')
-    self._cursor.execute(signal_load_mark_batch)
-    print('signal_dim_add_new_load:')
-    self._cursor.execute(signal_dim_add_new_load)
-    print('geo_dim_add_new_load:')
-    self._cursor.execute(geo_dim_add_new_load)
-    print('signal_history_load:')
-    self._cursor.execute(signal_history_load)
-    print('signal_latest_load:')
-    self._cursor.execute(signal_latest_load)
-    print('signal_load_delete_processed:')
-    self._cursor.execute(signal_load_delete_processed)
-    print("done.")
+    logger = get_structured_logger("run_dbjobs")
+    import time
+    time_q = [time.time()]
+
+    try:
+      self._cursor.execute(signal_dim_add_new_load)
+      time_q.append(time.time())
+      logger.debug('signal_dim_add_new_load', rows=self._cursor.rowcount, elapsed=time_q[-1]-time_q[-2])
+
+      self._cursor.execute(geo_dim_add_new_load)
+      time_q.append(time.time())
+      logger.debug('geo_dim_add_new_load', rows=self._cursor.rowcount, elapsed=time_q[-1]-time_q[-2])
+
+      self._cursor.execute(signal_history_load)
+      time_q.append(time.time())
+      logger.debug('signal_history_load', rows=self._cursor.rowcount, elapsed=time_q[-1]-time_q[-2])
+
+      self._cursor.execute(signal_latest_load)
+      time_q.append(time.time())
+      logger.debug('signal_latest_load', rows=self._cursor.rowcount, elapsed=time_q[-1]-time_q[-2])
+
+      self._cursor.execute(signal_load_delete_processed)
+      time_q.append(time.time())
+      logger.debug('signal_load_delete_processed', rows=self._cursor.rowcount, elapsed=time_q[-1]-time_q[-2])
+    except Exception as e:
+      raise e
```

Comment:
Should this exception be added to the logger so it's in the log file?

Reply:
well...

this method is only really called by `insert_or_update_batch` which in turn will again catch and re-raise this exception, and it states "rollback is handled in csv_to_database; if you're calling this yourself, handle your own rollback".

also, instantiating the [structured_logger](https://github.com/cmu-delphi/delphi-epidata/blob/v4-srrpp-migrations/src/acquisition/covidcast/logger.py#L21) registers a handler by default that should log any uncaught exceptions.

looking at this now (and unless im missing something), its really just a no-op to catch an unspecific exception and simply re-raise it.  the same situation effectively exists in the above mentioned `insert_or_update_batch`, but its presence coupled with the comment hints to a maintainer that exception handling is not an afterthought and is being dealt with elsewhere.

and to be fair, i cribbed the pattern in this method from @krivard 's version [here](https://github.com/cmu-delphi/delphi-epidata/blob/krivard/v4-rpp-mergeddim-leftjoin/src/acquisition/covidcast/database.py#L305) (because im currently in the middle of incorporating lots of her testing fixes from that branch into this one).  she probably had a reason for this that got lost along the way; perhaps other code was removed, like an Exception subclass case that was handled differently, or a finally block.

SO i think the correct thing to do here is to remove these no-op catch+reraise block boundaries, BUT i am going to leave them for now because they are harmless and we can discuss their efficacy later (or preferably sooner!).  i made a note to address this that i will either act on in the very near future, either by fixing it or turning it into a github issue.  i also made a note for some things i noticed in the logger class while investigating this, and it will have 

im not sure why i wrote such a novel for this.  ¯\\\_(ツ)_/¯

Reply:
> i think the correct thing to do here is to remove these no-op catch+reraise block boundaries, BUT i am going to leave them for now because they are harmless and we can discuss their efficacy later (or preferably sooner!)

+1, agree on both counts